src/Stripe.net/Services/EphemeralKeys/StripeEphemeralKeyCreateOptions.cs

```diff
@@ -9,5 +9,7 @@ public class StripeEphemeralKeyCreateOptions
     {
         [JsonProperty("customer")]
         public string CustomerId { get; set; }
+
+        public string StripeVersion { get; set; }
```

Comment:
`[JsonIgnore]`? I don't know how the serializer is configured, but otherwise it might include the field in the serialized JSON.

Reply:
yeah it works without it but it does not hurt to be clearer